selinux/ceph-installer.fc

```diff
@@ -0,0 +1,11 @@
+/usr/bin/gunicorn_pecan		--	gen_context(system_u:object_r:ceph_installer_exec_t,s0)
```

Comment:
I thought we were going to create a wrapper script for gunicorn and label that? The `/usr/bin/gunicorn_pecan` utility could be used elsewhere, not just ceph-installer